content/300-guides/200-deployment/201-serverless/150-deploy-to-azure-functions.mdx

```diff
@@ -1,640 +1,42 @@
-```
-
-Here's a breakdown of the command's parameters:
-
-- `--resource-group` adds the database to the resource group created in step 3
-- `--server` sets the Azure SQL database server to create it in
-- `--name` sets the name of the database
-- `--service-objective` sets the database's service tier that [determines the cost](https://azure.microsoft.com/en-us/pricing/details/sql-database/single/).
-
-## 6. Create a firewall rule to allow local access to the database
-
-In this step, you add two firewall rules:
-
-- Rule 1: Allow remote access from your local computer's public IP to the Azure SQL database. This is necessary so that you can create the database schema and use the database to test locally.
-- Rule 2: Allow access to the Azure SQL database from Azure Functions.
-
-### 6.1. Allow access from your local computer
-
-### Steps
-
-Determine your public IP with the following command:
-
-```terminal copy
-curl ifconfig.me
-```
-
-Copy the IP from the output and run the following command. Replace `YOUR_PUBLIC_IP` with the IP address and `UNIQUE_DB_SERVER_NAME` with the name from step 4.
-
-```terminal copy wrap
-az sql server firewall-rule create --resource-group prisma-azure-example --server UNIQUE_DB_SERVER_NAME --name allow-local-access --start-ip-address YOUR_PUBLIC_IP --end-ip-address YOUR_PUBLIC_IP
-```
-
-### Result
-
-The command outputs information similar to the following:
-
-```json no-copy
-{
-  "endIpAddress": "YOUR_PUBLIC_IP",
-  "id": "/subscriptions/YOUR_SUBSCRIPTION_ID/resourceGroups/prisma-azure-example/providers/Microsoft.Sql/servers/prisma-db/firewallRules/allow-local-access",
-  "kind": "v12.0",
-  "location": "Germany West Central",
-  "name": "allow-local-access",
-  "resourceGroup": "prisma-azure-example",
-  "startIpAddress": "YOUR_PUBLIC_IP",
-  "type": "Microsoft.Sql/servers/firewallRules"
-}
-```
-
-### 6.2 Allow access from Azure Functions
-
-To allow applications hosted inside Azure to connect to your SQL server, you must enable Azure connections. To enable Azure connections, you set up a firewall rule with starting and ending IP addresses set to `0.0.0.0`.
-
-### Steps
-
-Create the rule with the following command:
-
-```terminal copy wrap
-az sql server firewall-rule create --resource-group prisma-azure-example --server UNIQUE_DB_SERVER_NAME --name allow-function-access --start-ip-address 0.0.0.0 --end-ip-address 0.0.0.0
-```
-
-### Result
-
-The command outputs information similar to the following:
-
-```json no-copy
-{
-  "endIpAddress": "0.0.0.0",
-  "id": "/subscriptions/YOUR_SUBSCRIPTION_ID/resourceGroups/prisma-azure-example/providers/Microsoft.Sql/servers/UNIQUE_DB_SERVER_NAME/firewallRules/allow-function-access",
-  "name": "allow-function-access",
-  "resourceGroup": "prisma-azure-example",
-  "startIpAddress": "0.0.0.0",
-  "type": "Microsoft.Sql/servers/firewallRules"
-}
-```
+- An existing function app project with Prisma
```

Comment:
Could we rephrase this?
```suggestion
- An existing Azure Function project with Prisma
```
or something similar

Reply:
A function app and Azure function are two separate entities.

A project that has one or more functions is a "function app"

Reply:
Naming will be the death of me.